### PR TITLE
Fix project files

### PR DESCRIPTION
**Pull request by livecodefraser**

### CODE REVIEW

File: engine/Android.mk [hunk 1]
```diff
@@ -2,7 +2,7 @@ LOCAL_PATH := $(call my-dir)
 
 include $(CLEAR_VARS)
 
-DERIVED_SRC = ../_cache/mac/Debug/kernel.build/DerivedSources
+DERIVED_SRC = ../_cache/mac/Release/kernel.build/DerivedSources
```
Comment (peter-b) [inline engine/Android.mk:5]: This change doesn't look very relevant...?

Comment (livecodefraser) [inline engine/Android.mk:5]: Well, some of the Android makefiles referenced the Release sources and some the Debug...

Comment (peter-b) [inline engine/Android.mk:5]: Ouch

**Comment by peter-b** [comment]
I'm fairly surprised that there wasn't more horrible brokenness than that in my Windows code, given that I hadn't even compiled it! :stuck_out_tongue_winking_eye:

Reviewed-by: Peter TB Brett peter@livecode.com

**Pull request merged**